### ISSUE TITLE
MongoDB Java driver should be updated

### ISSUE THREAD

**Issue by GoogleCodeExporter** [epermana/tungsten-replicator#1102]
```
The latest Tungsten version as of 2015 March 6 is 3.0.1.

It still uses mongo-2.5.3.jar, which was released on 07-Apr-2011 (4 years ago!):
http://search.maven.org/#search%7Cgav%7C1%7Cg%3A%22org.mongodb%22%20AND%20a%3A%2
2mongo-java-driver%22

This driver doesn't work with newer versions of MongoDB.

When using Tungsten with MongoDB 2.4.9 on Ubuntu 14.04, when installing this 
error will be displayed:

ERROR >> test_ubuntu_mongodb >> At least one replication service has 
experienced an error
#####################################################################
# Commitment failed
#####################################################################
#####################################################################
# Errors for test_ubuntu_mongodb
#####################################################################
ERROR >> test_ubuntu_mongodb >> At least one replication service has 
experienced an error

"trepctl status" shows this pendingExceptionMessage:
Unable to start replication service due to underlying error: 
com.mongodb.MongoException$Network: can't call something

When I manually replace mongo-2.5.3.jar with version 2.13.0 (released on 
29-Jan-2015), unload, then load Tungsten again, Tungsten works!

In short, please update the MongoDB Java driver.
```

Original issue reported on code.google.com by `ngocdaot...@gmail.com` on 6 Mar 2015 at 7:54
